base/server/python/pki/server/subsystem.py

```diff
@@ -2255,9 +2256,6 @@ def run(self,
         if capture_output:
             stdout = subprocess.PIPE
             stderr = subprocess.PIPE
```

Comment:
If `capture_output=True` and `stdout` assigned to something different is it OK to overwrite?